combine/combine_test.go

```diff
@@ -0,0 +1,216 @@
+package combine
+
+import (
+	"bufio"
+	"compress/gzip"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestCombine(t *testing.T) { //nolint:gocognit
+	Convey("The function combines the files", t, func() {
```

Comment:
Having decided on the initial public API, replace these tests with tests for the public functions we've decided on.